smoke-tests-otel-starter/src/main/java/io/opentelemetry/spring/smoketest/RuntimeHints.java

```diff
@@ -14,5 +16,15 @@ public class RuntimeHints implements RuntimeHintsRegistrar {
   public void registerHints(
       org.springframework.aot.hint.RuntimeHints hints, ClassLoader classLoader) {
     hints.resources().registerResourceBundle("org.apache.commons.dbcp2.LocalStrings");
+
+    // To remove from Spring Boot 3.2.3 release:
+    // https://github.com/spring-projects/spring-data-commons/issues/3025
```

Comment:
do we want to remove this when SpringBoot 3.2.3 is released, or do we want to keep it for users who may still be using 3.2.2?

Reply:
The problem is related to the application used to test the OpenTelemetry starter (without the OpenTelemetry starter code, the application fails with the same error). The workaround is applied in the OpenTelemetry starter smoke test project.

Reply:
> The workaround is applied in the OpenTelemetry starter smoke test project.

oops I missed this part, thx!